lib/questions.rb

```diff
@@ -208,6 +247,7 @@ def your_birthday_is_on_a_friday_in_the_year(birthday)
 # and 1 that is 4 letters long. Return it as a hash in the format
 # word_length => count, e.g. {2 => 1, 3 => 5, 4 => 1}
 def count_words_of_each_length_in_a_file(file_path)
+  File.new(file_path).read.split(/\W+/).inject({}){|accum, word| accum.has_key?(word.length) ? accum.merge({ word.length => accum[word.length]+1 }) : accum.merge({ word.length => 1}) }
```

Comment:
Line is too long. [184/80]<br>Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator '+'.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space inside } missing.